sr_blockly_blocks/generators/python/scripts/initialise.py

```diff
@@ -9,6 +9,8 @@
     hand_commander = SrHandCommander(name=side + "_hand")
 elif dropdown_commander == 'robot_commander':
     robot_commander = SrRobotCommander(name=side + "_arm_and_hand")
+elif dropdown_commander == 'hand_h_and_arm':
+    robot_commander = SrRobotCommander ("right_arm_and_hand", prefix="H1_")
```

Comment:
Oh, ok, get you. Changing side to make sense would mean a lot of other changes...